scripts/publish_snapshot.ts

```diff
@@ -0,0 +1,10 @@
+import { runPublish } from './publish_runner.js';
+
+const tag = process.argv[2];
+
+await runPublish({
+  includeGitTags: false,
+  useLocalRegistry: false,
+  snapshotRelease: true,
```

Comment:
since tag only applies for snapshot releases, maybe we should just have an optional `snapshotTag?: string` option